.stylelintrc.json

```diff
@@ -1,23 +1,36 @@
 {
-  "extends": ["stylelint-config-standard", "stylelint-prettier/recommended"],
-  "plugins": ["stylelint-order", "stylelint-scss"],
+  "extends": [
```

Comment:
copied from Vanilla, there is no easy way to share it between repos (without publishing it as stylelint config npm package)